core/src/main/scala/kafka/tools/ExportZkOffsets.scala

```diff
@@ -52,28 +52,29 @@ object ExportZkOffsets extends Logging {
 
     val zkConnectOpt = parser.accepts("zkconnect", "ZooKeeper connect string.")
                             .withRequiredArg()
+                            .describedAs("url(s) for the zookeeper connection")
                             .defaultsTo("localhost:2181")
                             .ofType(classOf[String])
```

Comment:
https://cwiki.apache.org/confluence/display/KAFKA/System+Tools#SystemTools-ExportZookeeperOffsets
In the above link, it shows that zkconnect is a required option but it also has a defaultsTo value set. So the defaultsTo value does not work if we do not specify zkconnect. Per Vahid's suggestion, I went ahead and made this a regular option(not a required option) with the defaultsTo value and instead made the output file required (this way it is consistent with ImportZkOffsets). Thanks Vahid